### PR TITLE
[WIP] rename copy module due to namespace collision with global namespace

### PR DESCRIPTION
**Pull request by ebroecker**

### CODE REVIEW

File: src/canmatrix/__init__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 import canmatrix.formats as formats
 import canmatrix.cancluster as cancluster
 import canmatrix.convert as convert
-import canmatrix.copy as copy
+import canmatrix.cmcopy as cmcopy
```
Comment (altendky) [inline src/canmatrix/__init__.py]: If anything is needed I believe `import canmatrix.copy` is sufficient.  But really if someone wants a submodule they should import it themselves, not expect importing it's parent to import it.

File: src/canmatrix/canmatrix.py [hunk 2]
```diff
@@ -44,11 +41,12 @@
     from itertools import zip_longest as zip_longest
 except ImportError:
     from itertools import izip_longest as zip_longest
+
 from itertools import chain
 import struct
 
 from past.builtins import basestring
-import copy 
+from . import cmcopy
```
Comment (altendky) [inline src/canmatrix/canmatrix.py]: Just `import canmatrix.copy` I would think?  `cmcopy` is used once.

File: src/canmatrix/convert.py [hunk 3]
```diff
@@ -31,7 +31,7 @@
 sys.path.append('..')
 import canmatrix.formats
 import canmatrix.canmatrix as cm
-import canmatrix.copy as cmcp
+import canmatrix.cmcopy as cmcp
```
Comment (altendky) [inline src/canmatrix/convert.py]: We use `cmcp` five times, can we really not write `canmatrix.copy` instead?

**Comment by ebroecker** [comment]
@wow - I could have bet that I tried this before, but seems it works now - the cleanest approach for the namespace issue...

**Pull request merged**